packages/serverless-nextjs-component/serverless.js

```diff
@@ -449,14 +452,19 @@ class NextjsComponent extends Component {
 
     const defaultEdgeLambdaPublishOutputs = await defaultEdgeLambda.publishVersion();
 
+    const defaultCloudfrontInputs =
+      (inputs.cloudfront && inputs.cloudfront.defaults) || {};
     const cloudFrontOutputs = await cloudFront({
       defaults: {
         ttl: 0,
         allowedHttpMethods: ["HEAD", "GET"],
+        ...defaultCloudfrontInputs,
```

Comment:
Cloud we not follow the same pattern as above and pass `...defaultCloudfrontInputs` at the end of the object so you don't need the `...defaultCloudfrontInputs.forward` bit?

Reply:
No. Unfortunately, the spread operator is not that clever in JS.  If we did this, and no *forward* key was defined, it would result in no forward section being present since it would get deleted by `... defaultCloudfrontInputs`

Reply:
Hmmmm - you are making me doubt this now... if no forward section is defined, then I think you are correct, it will work... but this is easy to test... I will try it, and if the unit tests pass, then it works like you say ;)

Reply:
Ah, okay, I know why I did it now...

So if you have this config:

```
cloudfront:
  defaults:
    forward:
       headers: [ blah ]
```
Then, the forward section would be totally replaced (and not augmented) by headers. I thought that this would become a nightmare with people making mistakes and then creating issues. So the way I have it now, it always tries to augment the forward section, never replaces it.

Reply:
Makes sense! Let's keep it as you have it then.